### PR TITLE
Merge sierra and mets

### PR DESCRIPTION
**Pull request by alicefuzier**

### CODE REVIEW

File: pipeline/merger/src/main/scala/uk/ac/wellcome/platform/merger/rules/sierramets/SierraMetsWorkPairMerger.scala [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  Location,
+  TransformedBaseWork,
+  Unidentifiable,
+  UnidentifiedRedirectedWork,
+  UnidentifiedWork
+}
+import uk.ac.wellcome.platform.merger.model.MergedWork
+import uk.ac.wellcome.platform.merger.rules.{MergerRule, WorkPairMerger}
+
+trait SierraMetsWorkPairMerger extends WorkPairMerger {
```
Comment (warrd) [inline pipeline/merger/src/main/scala/uk/ac/wellcome/platform/merger/rules/sierramets/SierraMetsWorkPairMerger.scala]: Might help having a scaladoc comment on this trait, with a brief summary of the merging logic within.

Comment (alicefuzier) [reply pipeline/merger/src/main/scala/uk/ac/wellcome/platform/merger/rules/sierramets/SierraMetsWorkPairMerger.scala]: I've just added it

File: pipeline/merger/src/main/scala/uk/ac/wellcome/platform/merger/rules/sierramets/SierraMetsWorkPairMerger.scala [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+import uk.ac.wellcome.platform.merger.model.MergedWork
+import uk.ac.wellcome.platform.merger.rules.{MergerRule, WorkPairMerger}
+
+/**
+  * A METS work contains information relating to the digitisation of a Sierra work.
+  * Currently that amounts to a digital location on the item which contains
+  * the url to the iiif-presentation API and the license associated with it.
+  *
+  * If we have a Sierra and a METS work each one with a single item,
+  * we merge the digital location of the METS work into the locations of the Serra work.
```
Comment (warrd) [inline pipeline/merger/src/main/scala/uk/ac/wellcome/platform/merger/rules/sierramets/SierraMetsWorkPairMerger.scala]:
```suggestion
  * we merge the digital location of the METS work into the locations of the Sierra work.
```

Comment (alicefuzier) [reply pipeline/merger/src/main/scala/uk/ac/wellcome/platform/merger/rules/sierramets/SierraMetsWorkPairMerger.scala]: Sorry, I fixed this straight in master after merging this: https://github.com/wellcometrust/catalogue/commit/e09e1179b2f736a07a1d44a9dad18e8d5430b16a

Thanks!

**Pull request merged**